base/src/test/java/info/metadude/kotlin/library/c3media/ProductionApiTest.kt

```diff
@@ -1,6 +1,11 @@
 package info.metadude.kotlin.library.c3media
 
-import info.metadude.kotlin.library.c3media.models.*
+import info.metadude.kotlin.library.c3media.models.AspectRatio
+import info.metadude.kotlin.library.c3media.models.Conference
+import info.metadude.kotlin.library.c3media.models.Event
+import info.metadude.kotlin.library.c3media.models.Language
+import info.metadude.kotlin.library.c3media.models.MimeType
+import info.metadude.kotlin.library.c3media.models.Recording
```

Comment:
Can you please check if the imports fold to `import info.metadude.kotlin.library.c3media.models.*` if you use the default IntelliJ settings. It does on my machine.